### ISSUE TITLE
Warning: Mismatched Versions

### ISSUE THREAD

**Issue by EnamyX** [madMAx43v3r/chia-gigahorse#110]
Why, with a new version of the wallet and gigahorses with the same name, does it still write about the version mismatch?

Warning: Mismatched Versions
The Chia application and its services are using different versions. Some functionality may not be behave properly as a result.
Chia services version: 1.8.0.giga10
Chia application version: 1.8.0
It is recommended that you quit the Chia application and stop all Chia services.

Why are you giving the gigahorse such a name when you can give it a name like a wallet to eliminate this problem???

**Comment by madMAx43v3r** [comment]
To see that you are running Gigahorse and not normal Chia. Just ignore this error if the version numbers are matching.

**Comment by EnamyX** [comment]
> To see that you are running Gigahorse and not normal Chia. Just ignore this error if the version numbers are matching.

it's hard to ignore it when you set it to autorun when you start your computer. farming doesn't continue until you press the button...

**Comment by madMAx43v3r** [comment]
Farming does start without the GUI, it's just the GUI not loading automatically.

**Comment by EnamyX** [comment]
> Farming does start without the GUI, it's just the GUI not loading automatically.

Until I press the button, it says to me that the node is not synchronizing and the circle of the full node is red. While the node is not synchronized, farming does not work.

**Comment by madMAx43v3r** [comment]
That's just the GUI display, all the work is done by the background services. You can check via `chia.exe farm summary` in terminal.

**Comment by EnamyX** [comment]
> That's just the GUI display, all the work is done by the background services. You can check via `chia.exe farm summary` in terminal.

I use windows, there is no terminal :)

**Comment by madMAx43v3r** [comment]
Yes of course, double click on `start_farmer.cmd` in the gigahorse folder.

**Comment by EnamyX** [comment]
> Yes of course, double click on `start_farmer.cmd` in the gigahorse folder. That will start gigahorse, the you can type `chia.exe farm summary` there.

Ok, I understand, Thank you.

Tell me, does the power of the farm depend on the power of the video card?
For example, if I have 5000 c8 plots, will a 1660s video card be enough for me? or do I need a more powerful card?

**Comment by madMAx43v3r** [comment]
1660 super is enough for ~350 TB C8 yes

**Comment by EnamyX** [comment]
> 1660 super is enough for ~350 TB C8 yes

Ok? sps

**Comment by MLKOBAYASHI** [comment]
I see the same warning message of the same problem occurring again in the new version (1.8.2) but using normal Chia.

![XCH](https://github.com/madMAx43v3r/chia-gigahorse/assets/121802533/0de11a40-2e4b-49b5-97a9-227026853695)

Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz   2.70 GHz, 32,0 GB, Windows 10 Pro, Version 22H2, SO Compilation 19045.3324, Windows Feature Experience Pack 1000.19041.1000.0 just farming nowadays.

Flexpool.io dashboard

![image](https://github.com/madMAx43v3r/chia-gigahorse/assets/121802533/d6da05ec-9138-439c-9a2b-e01971a1065f)

Everything was going well until I updated in the first week of release of version 1.8.2 and from that day onwards performance has been precarious, waiting for perhaps a correction in version 1.8.3 that never happened so far, within a deadline never before occurred.

What to do until then? Does anyone have any suggestions to remedy this?

Thank you!

**Comment by madMAx43v3r** [comment]
make sure to properly stop everything when updating, via `chia stop all -d`
